### DISCUSSION TITLE
v4.11.0

### DISCUSSION THREAD

**Discussion by stefanzweifel** [stefanzweifel/git-auto-commit-action#157] [Announcements]
### Added
- Add `status_options` and `add_options` option [#156](https://github.com/stefanzweifel/git-auto-commit-action/pull/156)

<hr /><em>This discussion was created from the release <a href='https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v4.11.0'>v4.11.0</a>.</em>

**Comment by Difinit** [comment]
- [ ]
